demo/kubernetes/rook-client/rook-client.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rook-client
+spec:
+  containers:
+  - name: rook-client
+    image: quay.io/rook/k8s-demo
+    imagePullPolicy: IfNotPresent
+    command: ["sleep", "36500d"]
```

Comment:
Yes, we would have to package the pause binary in the container at build time. I was just pointing out that this is an established pattern and that the binary already exists.